pkg/agent/route/route_linux.go

```diff
@@ -1198,6 +1212,37 @@ func (c *Client) initNodeNetworkPolicy() {
 	}
 }
 
+func (c *Client) initWireguard() {
+	wireguardPort := intstr.FromInt(c.wireguardPort)
+	antreaInputChainRules := []string{
+		iptables.NewRuleBuilder(antreaInputChain).
+			SetComment("Antrea: allow input wireguard packets").
+			MatchTransProtocol("udp").
```

Comment:
I have added some consts in package iptables. You can use `iptables.ProtocolUDP`.

Reply:
updated.